Node.js TypeScript quickstart for Managed Redis (#66)
Azure-Samples/azure-cache-redis-samples: quickstart/nodejs/README.md

# Node.js Redis Sample

> This is a sample application demonstrating how to use Azure Cache for Redis with a Node.js application.

Reviewer: Recommend replacing all occurrences of "Azure Cache for Redis" with "Azure Managed Redis"

## Prerequisites

- Node.js LTS
- Azure subscription

> - Azure Managed Redis instance with appropriate Entra ID permissions

Reviewer: ```suggestion
- Azure Managed Redis instance with Entra ID authentication enabled and appropriate data access policy assignments

```

## Getting Started

1. Clone the repository:

   ```bash
   git clone https://github.com/Azure-Samples/azure-cache-redis-samples.git
   cd azure-cache-redis-samples/quickstart/nodejs
   ```

2. Install dependencies:

   ```bash
   cd quickstart/nodejs
   npm install
   ```

3. Copy `sample.env` to a `.env` file and add your Azure Cache for Redis endpoint:

   ```env

>    AZURE_MANAGED_REDIS_HOST_NAME="rediss://<RESOURCE_ENDPOINT>:10000"

Reviewer: ```suggestion
   AZURE_MANAGED_REDIS_ENDPOINT="rediss://<RESOURCE_HOST_NAME>:10000"

```

# Node.js Redis Sample

This is a sample application demonstrating how to use Azure Managed Redis with a Node.js application.

## Prerequisites

- Node.js LTS
- Azure subscription
- Azure Managed Redis instance with Entra ID authentication enabled and appropriate data access policy assignments

## Getting Started

1. Clone the repository:

   ```bash
   git clone https://github.com/Azure-Samples/azure-cache-redis-samples.git
   cd azure-cache-redis-samples/quickstart/nodejs
   ```

2. Install dependencies:

   ```bash
   cd quickstart/nodejs
   npm install
   ```

3. Copy `sample.env` to a `.env` file and add your Azure Managed Redis endpoint:

   ```env

>    AZURE_MANAGED_REDIS_ENDPOINT="rediss://<RESOURCE_HOST_NAME>"

Reviewer: ```suggestion
   AZURE_MANAGED_REDIS_ENDPOINT="<REDIS_HOST_NAME>:<PORT>"

```
